lib/msal-react/src/MsalContext.tsx

```diff
@@ -10,47 +11,12 @@ export interface IMsalContext {
     state: MsalState;
 }
 
-// Stubbed context implementation
+/*
+ * Stubbed context implementation
+ * Only used when there is no provider, which is an unsupported scenario
+ */
 const defaultMsalContext: IMsalContext = {
-    instance: {
-        // Msal methods
-        acquireTokenPopup: () => {
-            return Promise.reject();
-        },
-        acquireTokenRedirect: () => {
-            return Promise.reject();
-        },
-        acquireTokenSilent: () => {
-            return Promise.reject();
-        },
-        getAllAccounts: () => {
-            debugger;
-            return [];
-        },
-        getAccountByUsername: () => {
-            /*
-             * TODO: getAccountByUsername should have a return type of `AccountInfo | null`
-             * Also should prevent possible null reference exception if no account matches the username
-             * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/src/app/PublicClientApplication.ts#L466
-             */
-            return (null as unknown) as AccountInfo;
-        },
-        handleRedirectPromise: () => {
-            return Promise.reject(null);
-        },
-        loginPopup: () => {
-            return Promise.reject();
-        },
-        loginRedirect: () => {
-            return Promise.reject();
-        },
-        logout: () => {
-            return Promise.reject();
-        },
-        ssoSilent: () => {
-            return Promise.reject();
-        },
-    },
+    instance: new PublicClientApplication({}),
```

Comment:
What happens if one of the APIs are invoked on this default instance?

Reply:
What is the scenario where that would happen? Imo the default should never be used. The reason I removed the stubbed functions was that I don't want that to be another thing we need to maintain when we add/remove APIs from msal-browser.

Reply:
I think if you use the MsalContext but don't provide an instance. One thing I would be concerned about are any side effect that are caused by instantiating `PublicClientApplication`. 

Also, I don't see maintaining the stub implementation as a big deal, given we would already need to update the wrapper instance if new APIs are added.

Reply:
I don't think this is ever instantiated if you have a Provider above and I think using the Context without a Provider should be unsupported. I was hoping to also get rid of the wrapper instance once the event API is added...

Reply:
Agreed to have the stubbed instance live in `msal-browser`, allowing it to be used by other framework libraries and forcing us to update the stubbed instance when we update the PCA interface.